test/utils.ts

```diff
@@ -0,0 +1,20 @@
+import { FhirResourceType } from '@projecttacoma/node-fhir-server-core';
+import { Connection } from '../src/db/Connection';
+
+const createTestResource = async (data: fhir4.FhirResource, resourceType: FhirResourceType) => {
+  const collection = Connection.db.collection<fhir4.FhirResource>(resourceType);
+  await collection.insertOne({ ...data });
+};
+
+export async function cleanUpDb() {
+  await Connection.db.dropDatabase();
+  await Connection.connection?.close();
+}
+
+export async function testSetup(testFixtureList: fhir4.FhirResource[]) {
```

Comment:
^ I agree, `testDatabaseSetup` sounds good, or maybe `setupTestDatabase` (personally when I see `testSetup`/`testDatabaseSetup` I associate the function with testing the setup rather than getting set up for the tests so I prefer the verb coming first but that's just me)

Reply:
`testDatabaseSetup` sounds nice :)